entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+image=$1
+report=$2
+patched_tag=$3
+
+# parse image into image name and image tag
+image_no_tag=$(echo "$image" | cut -d':' -f1)
+old_tag=$(echo "$image" | cut -d':' -f2)
+
+# new patched image tag
+new_tag="$old_tag$patched_tag"
```

Comment:
what do you think about just specifying a new tag instead of appending? this way user can specify anything